bookkeeper-server/src/main/java/org/apache/bookkeeper/client/api/CreateBuilder.java

```diff
@@ -100,6 +101,16 @@
      * @return the builder itself
      */
     CreateBuilder withWriteFlags(EnumSet<WriteFlag> writeFlags);
+/**
```

Comment:
a space line for this line. and wrong indent.

Reply:
done